initial-cluster-setup/gp-initial-cluster-setup/values/values-play.yaml

```diff
@@ -78,6 +78,13 @@ applications:
             value: "true"
           - name: "htpasswd.data"
             value: "AgAKDrNtxXyshcqK3/NHGmTYKQZuHG1JrRDdO04XQxfIIQsCQAKbhsgvV+ML2tqIG7F7qg+4zNVM3Z6fLtNAnyHZh/acTVN1V9XxB1eW5gNPwooVVvET5Nr2ivD3n5vIuhASP/bWzrU4w92T7Ha3Ghl8WofypDwYuT6wIvSL3FcE+PelJYVJkyufQ9WNM5vmaDIEEkzWl5S4CJikY1RlGPm7SkkXFGRymPVKj8/nDrybjFUTln/HogMKiGNnwOg36q52HTAikY9jPGowJcfFRgeMOFgh3yu0RimxgG0ZeheF4DrxxC2z8oUrLtot7tM62Fzy/58hl5cNtU780qW4tu6MdeOlvhP5WBdDB81QBORxsp/+ybPjxtiZFvLB+FaMJkzt5K5Z0I0EIhWeNetHj9bqb46c/ZfutHOxuJzwlBgrGBSBvmstO7nVax2f2kwP0GRurcIBhLXfieROU4NGsUJcQnKavXqFpxj56uAkB3Fa7nmfAp7WGLlt0iV9dXkggCm7N/ISHVuQQkWhuZtqZ5cpriQbjac/oCHKo4UV6DUO3dQnhFq3JT85tnL2nJpPg/kzYeHgjlKS1Vnh9tts4n9yjqTzkV5xSXJ15B/gOHLbtWW12Tf25ly3bHqbQ8wmQm4C6egLT+Ffw4ldLfIYxdnIZCcXdVOZWmD7mOKbdDI3o7rXefc6gpUnOxE+e1/D5IZq3GH0szk4aK+izx0pIv2ILKrhnRHMGgpbviaWaNcvmlG9UMQWo0EcJ2Rxs8eCf4RsbEgosXF0mwQfcyX0cj6Gj0mEB9s="
+          - name: "ldap.bindPassword"
+            value: "AgBAw7IrV751kSjQwhlb8L9ADP6Y3IYhI+t8f1IGI3y3/TmJDNxNGepfHAGDQsKAijRbhSP7SAYhcyMnq8buVgchf7VA4QPnB59meJZovLsrjx0GfPGrXVcCihqpfzM0VA6yGYuhll06AcUfHwzG5fiqWCXKjRoZt3iQTefNSWWJRk/2DZC2Vkq08ndOK7aVUlnoJKBeATnzT446zBShr/czxqp+AK3w0hynoxV8DeTZHOpnAuO5OKB7CzBkEEKRJk+/VB90HuI3ZbTiOuOJfVK/Pgc+1tjfy8C0zgnsrMIpckYQ0cbIX4EmWkz/OHMDxneGmHPeRSKwK0XBa7gbHv/s1W56512reWBZkV73pNCfPacyXMqXEF7E2bOhdAMVJEg7i3IrWIs+FHzKRSV1yzpUnGFwhAfAj2CfvuS4iudp8YWe255ebzYy5wZc6MNJ761zEzAzgzf8qkuRBSBCrysrF2VS/nSy6B7fLpyg2D0qmrzB8+owhQ9G5VKFE0PwLnUqnb2oDcLCUwsVMbeRXuv2vvGF+YO6Ytc6pAPAfAAB8LsgUoE0u3zLAcutU/36DP1YpeDzjLVQ+gtmfMPaI1z8ft9Ra/xa0KQ33M1YH7MC6UspU0JzPhX9GMSk6hMdSfAJOlOPJEtedeQYN8AMaDwd1DK8KCbNaWdjMQxL3whQw+vLhvFOaF6YRcKiYGfk/ieaVBIsQczOkOonKSzE5g=="
+          - name: "ldap.bindDN"
+            value: "AgAy3WkzeEsCYkAZDuwi/EsHJw1ELmlIHKkjd5h9flLqNmhLmThNiOoRykOomjxeTJOx5RXw8vMFjElpVkDw38I6NyHbhOLzFertcSL4kZdQ3crCVD7fCtpQF3mQXvqtt7/KOpQjoRhQHrwXzoCJ0R+85L7qt19Pu+7192eGDwXAe4NFsurObMJObI8uiO6bPAIo0zPnRJGwJhpbJZFDKrnx/7VioIbfmnQMQzu8Whibn7B0mPJ7GfimApNXLWn+EhDKg4/cmbsSLnoXqPV13OSIBg+rUi7am24AySYMt1bvUmoTNw1omxZqj1q18xjFaYzQXd70S9OlsPd9YNXtOsxByvZFsoqmF9/bgZc7Pp3Fq43szM8lD46JXWB51RGCQ38NRBwMj23+3ARt2DGVPuNauuSICYTWOWqM3LYqSoj4Jo5hJp1jYCb6DzFW1bXaHYz4zki92bMX2IWdo45yvwGgPt/Jnjf3ga4nacEia3EG+hZPnCCv2NU53M41xSPNL+E0V3vaq3/BBDKNoBQOP8JapaLdb3zfHUXNUvMa9wobno+RoePIwvHYbcfl6nxRd/557bpwW1z0z9XYXlmGg+ld3QMEeLz7vISGDMFAqB7yfJxjpumKdaEozmBpJyC12t2/VJXVsRtDuCPdPxeQEr2W3JzbrQQfxxPH/OKd76/OvN+jtVWlJAvib0XenmSi7GH2MbrWtQbXppoZ2kKL6OK6zbhk5TJy9vXENYdLTO+xRSvKIXmMENTakuhWGOXgRm8="
+          - name: "ldapUrl"
```

Comment:
siehe Anmerkung im PR in gp-helm-chart-development zu meiner Frage bezüglich ldapUrl